services/postgres.sh

```diff
@@ -1,11 +1,6 @@
 #!/bin/bash -e
 # Begin service ENV variables
-export SHIPPABLE_START_POSTGRES=true;
-export SHIPPABLE_POSTGRES_PORT=5432;
-export SHIPPABLE_POSTGRES_VERSION="9.6";
-export SHIPPABLE_POSTGRES_BINARY="/usr/lib/postgresql/$SHIPPABLE_POSTGRES_VERSION/bin/postgres";
-export SHIPPABLE_POSTGRES_CMD="sudo -u postgres $SHIPPABLE_POSTGRES_BINARY -c config_file=/etc/postgresql/$SHIPPABLE_POSTGRES_VERSION/main/postgresql.conf";
-
+source "$(dirname $0)/postgres_env.sh"
```

Comment:
I get the following warning from [shellcheck](https://github.com/koalaman/shellcheck):

````
source "$(dirname $0)/postgres_env.sh"
                  ^-- SC2086: Double quote to prevent globbing and word splitting.
````